src/Microsoft.AspNetCore.Razor.Test.Sources/CaseSensitiveTagHelperAttributeComparer.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace Microsoft.AspNetCore.Razor.Test.Internal
```

Comment:
@dougbu Seeing that it's currently all based off of sharing TagHelper test logic I think that sounds good. If we take that approach we should probably swap the package name too.

I think the namespace/package name swapping could done in a separate PR if you didn't want to do it right away (or you can do it here if you want). Leaving this as `Razor.Test.Internal` for now at least maintains the consistency.